Makefile

```diff
@@ -150,3 +150,14 @@ OPERATOR_SDK = $(shell which operator-sdk)
 endif
 endif
 
+.PHONY: sync-manifests
+sync-manifests:
```

Comment:
I've been bitten by rolling python3 code before. Can we use a container for podman so that we can control the actuall version that is used.

This can be a followup.

Reply:
Fixed